app/cli/internal/action/attestation_init.go

```diff
@@ -229,6 +229,9 @@ func (action *AttestationInit) Run(ctx context.Context, opts *AttestationInitRun
 		return "", err
 	}
 
+	// Resolve the hosted context information
+	action.c.ResolveHostedRunnerContext(ctx)
```

Comment:
why do we need to run this now? What has changed?

Reply:
`att init` vs `att push`. Which step do you think is best for running this?

Reply:
I think the init state ain't bad, but looking for your input guys.

Reply:
I think init it's fine, specially because eventually we should fail if the authentication is incorrect

Reply:
Great, thanks.